docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+
+services:
+  web:
+    container_name: web
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - '3000:3000'
+      - '5000:5000'
+    environment:
+      - NODE_ENV=production
+    volumes:
+      - '.:/app'
+  nginx:
+    build: ./nginx
+    container_name: nginx
+    ports:
+      - 85:80
+    depends_on:
+      - web
+  db:
+    image: mongo:4.1.8-xenial
```

Comment:
I think we should use an external db accessible from the url to be able to easily migrate to anther platform when necessary and mange

Reply:
The db shouldn't be shipped together with the app

Reply:
Despite being in docker,it's accessible from outsite... check the next line where I declared ports

Reply:
> The db shouldn't be shipped together with the app

Can I know why?🤔

Reply:
If it is the case, what would you suggest?
Using managed database from digital ocean which will increase additional cost or installing our own local database in the droplet?

Reply:
@oesukam, I just did some research on a dockerizing database in production...You have a good point., and I totally agree with you.

But this is not yet a production environment, it still a development environment, because we don't have users yet. Calling it development is abus de language. it's till development or quality assurance environment.

What I would suggest is to keep it like this , but next time before merging to master and going live we can move to managed database if we can afford the cost or we can install our own database in the droplet if we can afford the security issue it may raised.